Libraries/Opc.Ua.Client/INodeCache.cs

```diff
@@ -64,6 +67,11 @@ public interface INodeCache : INodeTable, ITypeTable
         /// </summary>
         IList<INode> FindReferences(ExpandedNodeId nodeId, NodeId referenceTypeId, bool isInverse, bool includeSubtypes);
 
+        /// <summary>
+        /// Returns the references of the specified nodes that meet the criteria specified.
+        /// </summary>
+        IList<INode> FindReferences(IList<ExpandedNodeId> nodeIds, NodeId referenceTypeId, bool isInverse, bool includeSubtypes);
+
```

Comment:
*1
For optimization reasons I suggest that we provide also the possibility to specify more than one ReferenceTypeId. The method signature  would be:
```
/// <summary>
/// Returns the references of the specified nodes that meet the criteria specified.
/// </summary>
IList<INode> FindReferences(IList<ExpandedNodeId> nodeIds, IList<NodeId> referenceTypeIds, bool isInverse, bool includeSubtypes);
```

Reply:
good idea...